playbooks/deploy-machines-setup.yml

```diff
@@ -10,27 +10,21 @@
 
   # Load common vars
   vars_files:
-  - vars/deploy-machines.yml
+    - vars/deploy-machines.yml
 
   roles:
-
     # Install Docker
     - role: geerlingguy.docker
       become: True
 
     # Secure the server
     - role: geerlingguy.security
       become: True
-  
+
   tasks:
+    - name: Check server OS compatibility
```

Comment:
same: `Include checking ...`

Reply:
renamed in https://github.com/SkynetLabs/ansible-playbooks/pull/238/commits/de6d344272248acf2313779c0dbc53758035699c